src/app/datagrid/selection/selection.ts

```diff
@@ -31,10 +31,31 @@ export class DatagridSelectionDemo {
     example = EXAMPLE;
     users: User[];
     selected: User[] = [];
+    toAdd: User[] = [];
+    toDelete: User[] = [];
+    toEdit: User;
 
     constructor(private inventory: Inventory) {
         inventory.size = 10;
         inventory.reset();
         this.users = inventory.all;
     }
+
+    onDelete(user: User) {
+        if (user) {
+            this.toDelete = [ user ];
+        } else {
+            this.toDelete = this.selected.slice();
+        }
+        return false;
```

Comment:
do we return false to stop the page from redirecting? Can we avoid this if we use a button instead of a link in the overflow menu?